inspirehep/dojson/utils.py

```diff
@@ -104,3 +104,38 @@ def strip_empty_values(obj):
         return type(obj)(new_obj)
     else:
         return obj
+
+
+def remove_duplicates_from_list(l):
+    """Remove duplicates from a list preserving the order.
+
+    We might be tempted to use the list(set(l)) idiom,
+    but it doesn't preserve the order, which hinders
+    testability."""
+    seen = set()
```

Comment:
@jalavik this guy here has the same question :P 
https://pythonadventures.wordpress.com/2013/10/31/remove-duplicates-from-a-list-and-keep-the-original-order-of-the-elements/